pommerman/cli/run_battle.py

```diff
@@ -65,6 +65,9 @@ def _run(record_pngs_dir=None, record_json_dir=None):
             actions = env.act(obs)
             obs, reward, done, info = env.step(actions)
 
+            if step is 1:
```

Comment:
pls delete.

Reply:
Yeah. It will not be there when it comes out of WIP. :D